src/Toolkit/Toolkit/UI/Controls/ScaleLine/ScaleLine.cs

```diff
@@ -277,7 +271,7 @@ public static double CalculateScale(Esri.ArcGISRuntime.Geometry.Polygon visibleA
 
             // Calculate the geodedetic distance between two points one 'pixel' apart
             var result = Geometry.GeometryEngine.DistanceGeodetic(center, centerOnePixelOver, Geometry.LinearUnits.Inches, Geometry.AngularUnits.Degrees, Geometry.GeodeticCurveType.Geodesic);
-            double distanceInInches = result.Distance;
+            double distanceInInches = result?.Distance ?? 0;
```

Comment:
DistanceGeodetic does not return null and never returned null -- it was just incorrectly annotated in YAML. This has been fixed in YAML and should carry over to dotnet-api in next update. So I'd just null-forgive `result` here.